### DISCUSSION TITLE
I have a question about custom indicators.

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [apache/skywalking#10900] [General]
I have built a cluster now, with three OAP nodes and one UI node, and I am going to connect to 360 applications. The apm-toolkit-meter plug-in is used to collect custom indicators, and I want to combine the data of the collected custom indicators with the link However, due to the large number of applications, I want to reduce the link collection rate, for example, to 1%. If this is the case, will the 1% link collection affect the collection of custom indicators, or custom Whether the correlation analysis of indicators and links will be affected.

**Comment by wu-sheng** [comment]
What is `custom indicators`? Please don't create new term.

**Reply by qigeqigeqi** [reply]
Metrics collected through the meter API

**Reply by wu-sheng** [reply]
What is `combine the data of the collected custom indicators with the link`? Your words are not clear in many places.

**Reply by qigeqigeqi** [reply]
I have set up a cluster now, with 3 OAP nodes and 1 UI node, and I am preparing to integrate with the 360 application. I am using Meter's API to collect custom metrics, and I want to associate the data reported by the Meter API with links to enable viewing of these reported data on the webpage along with the corresponding traces. However, due to a large number of applications, I want to reduce the link collection rate, for example, down to 1%. If that's the case, will the 1% link collection rate affect the accuracy of the data reported by the Meter API, or the accuracy of the association between the reported data and the related traces during correlation analysis?

**Reply by wu-sheng** [reply]
What is `link collection rate`? There isn't this term.

**Reply by qigeqigeqi** [reply]
sorry,it means SW_TRACE_SAMPLE_RATE = 100.

**Reply by wu-sheng** [reply]
That is a trace thing, it has nothing relative to your metrics.

**Comment by qigeqigeqi** [comment]
Sorry, one more question. If SW_TRACE_SAMPLE_RATE = 100, does it mean that every occurrence of an error in the Trace will be logged and stored, discarding some of the successful ones? In other words, does reducing the SW_TRACE_SAMPLE_RATE only decrease the amount of successful chain records saved, while every occurrence of an abnormal chain will definitely be saved?

**Reply by qigeqigeqi** [reply]
This is not mentioned in the docs, I'd like to make sure, thanks.

**Reply by wu-sheng** [reply]
This has nothing relative with the status. I can't see the connection.

**Comment by qigeqigeqi** [comment]
https://skywalking.apache.org/docs/main/next/en/setup/backend/trace-sampling/#recommendation
